spyderlib/config/main.py

```diff
@@ -553,6 +553,7 @@ def is_ubuntu():
               'editor/show/hide outline': "Ctrl+Alt+O",
               'editor/show/hide project explorer': "Ctrl+Alt+P",
               'editor/new file': "Ctrl+N",
+              'Editor/Open last closed':"Ctrl+Shift+N",

```

Comment:
I don't think we need a shortcut for this action. Besides, `Ctrl+Shift+<Letter>` are reserved to move to panes.